assets/components/chat.html

```diff
@@ -20,10 +20,20 @@ <h2 class="ui dividing header">Chat
           <div am-time-ago="{{$ctrl.getTime(chat)}}" class="date"></div>
         </div>
         <div class="extra text chat {{chat.type === 'user' ? '' : 'content minimal'}}">
+          <i class="plus icon" ng-if="chat.type === 'addVideo'"></i>
```

Comment:
That's what I was planning on doing, pulling this and the `{{chat.message}}` into a new component. I made a heart component in the other PR.